app/src/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+# Configuration file
+# key = value
+quarkus.datasource.db-kind=h2
+quarkus.datasource.jdbc.url=jdbc:h2:/Users/avibernstein/Documents/DataStoreSummer

```

Comment:
for now, it might be better to have this be a relative path to the home directory instead of Bernie's directory which would only work on his machine.